gameshim/stripper_mm.cpp

```diff
@@ -288,7 +290,7 @@ LevelInit_handler(char const *pMapName, char const *pMapEntities, char const *c,
         g_mapname.assign(stripper_nextfile.GetString());
         log_message("Loading %s for map \"%s\"", g_mapname.c_str(), pMapName);
     } else {
-        g_mapname.assign(pMapName);
+        g_mapname.assign(stripper_lowercase.GetInt() != 0 ? UTIL_ToLowerCase(pMapName) : pMapName);
```

Comment:
This will leak the value of UTIL_ToLowerCase (assign doesn't take ownership of the buffer)

You might fix this with something like:


    } else if (stripper_lowercase.GetInt()) {
        char* name = UTIL_ToLowerCase(pMapName);
        g_mapname.assign(name);
        delete[] name;
    } else {
        g_mapname.assign(pMapName);
    }

Reply:
Thanks for the remarks, corrected.